android/app/src/androidTest/java/PrnmTest.java

```diff
@@ -146,7 +146,7 @@ public void sharedSetup(Setup s) throws Exception {
         Address onChain = wallet.importAccount(Setup.SKs[s.Index]);
 
         String ethUrl = "ws://10.5.0.9:8545";
-        Config cfg = new Config(Setup.Aliases[s.Index], onChain, s.Adjudicator, s.Assetholder, ethUrl, "0.0.0.0", 5750);
+        Config cfg = new Config(Setup.Aliases[s.Index], onChain, s.Adjudicator, s.Assetholder, ethUrl, "0.0.0.0", 5750, s.TxFinalityDepth);
```

Comment:
Access `TxFinalityDepth` by the class `Setup` instead of an instance reference since this member is static.
```suggestion
        Config cfg = new Config(Setup.Aliases[s.Index], onChain, s.Adjudicator, s.Assetholder, ethUrl, "0.0.0.0", 5750, Setup.TxFinalityDepth);
```